conf/test.config

```diff
@@ -25,15 +25,15 @@ params {
     perform_shortread_qc                  = true
     perform_longread_qc                   = true
     shortread_qc_mergepairs               = true
-    perform_shortread_complexityfilter    = true
+    perform_shortread_complexityfilter    = false
     perform_shortread_hostremoval         = true
     perform_longread_hostremoval          = true
     perform_runmerging                    = true
     hostremoval_reference                 = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/genome/genome.fasta'
     run_kaiju                             = true
     run_kraken2                           = true
     run_bracken                           = true
-    run_malt                              = false
+    run_malt                              = true
```

Comment:
This will cause the tests to fail becaus eit runs out of memory, which is why we have a separate run for that.

Reply:
Shouldn't `run_malt` be false since we run the test manually?

Reply:
Correct - I don't think I was the one who suggested the change to be true, was I?!

Reply:
Might have been me 😬